src/style.css

```diff
@@ -6,6 +95,6 @@
     padding: 0;
     margin: -1px;
     overflow: hidden;
-    clip: rect(0,0,0,0);
+    clip: rect(0, 0, 0, 0);
```

Comment:
Thanks for updating!

Reply:
It's due to my auto-formatting on vs code 😅